guides/installation_guides/c++/mac/README.md

```diff
@@ -1,5 +1,34 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
+<b1> <b>Getting The Software</b> </b1> <br> <br>
+>There are many IDE's for MAC OS but here's a quick guide to install one of them i.e <b> CODEBLOCKS>. <br>
```

Comment:
Why is everything in a blockquote? Don't overuse formatting.

Reply:
okayy , i have updated it . you can check it now..
Thank you!